src/sentry/event_manager.py

```diff
@@ -588,10 +588,15 @@ def save(self, project, raw=False):
                 datetime=date,
             )
 
-        tsdb.incr_multi([
+        counters = [
             (tsdb.models.group, group.id),
             (tsdb.models.project, project.id),
-        ], timestamp=event.datetime)
+        ]
+
+        if release:
+            counters.append((tsdb.models.release, release.id))
+
+        tsdb.incr_multi(counters, timestamp=event.datetime)
```

Comment:
There's no test for any of these (that I could find, at least — there's one for frequencies, but it's xfailed.)